introduction.md

```diff
@@ -1,3 +1,6 @@
 hello, dojo
 my name is acronick
 my favorite Code School path is Ruby
+
+*Git
+*JS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?